core/src/css/core.scss

```
@@ -395,6 +395,23 @@ ion-input input::-webkit-date-and-time-value {
 }
 
 /**
+ * When moving focus on page transitions we call .focus() on an element which can
+ * add an undesired outline ring. This CSS removes the outline ring.
+ * We also remove the outline ring from elements that are actively being focused
+ * by the focus manager. We are intentionally selective about which elements this
+ * applies to so we do not accidentally override outlines set by the developer.
+ */
+[ion-last-focus],
+header[tabindex="-1"]:focus,
+[role="banner"][tabindex="-1"]:focus,
+main[tabindex="-1"]:focus,
+[role="main"][tabindex="-1"]:focus,
+h1[tabindex="-1"]:focus,
+[role="heading"][aria-level="1"][tabindex="-1"]:focus {
```

Could we do the following to lessen the amount of lines? Or is it more important to point out which elements are being targeted?

```suggestion
[ion-last-focus],
*[tabindex="-1"]:focus {
```

---

This changes which elements are going to be selected. The focus manager works on specific semantic elements/landmarks, but the proposed change would apply `outline: none` to any focused element that has `tabindex="-1"`. The particular elements that the focus manager targets is outlined on the docs: https://ionic-docs-git-focus-manager-docs-ionic1.vercel.app/docs/developing/managing-focus#content-types